src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -280,8 +280,17 @@ def get(self, request: Request, project: Project) -> Response:
                         )
                         if result["lineCoverage"] and result["codecovUrl"]:
                             result["codecovStatusCode"] = 200
+                            result["codecov"] = {
+                                "lineCoverage": result["lineCoverage"],
+                                "codecovUrl": result["codecovUrl"],
+                                "statusCode": result["codecovStatusCode"],
+                            }
```

Comment:
`lineCoverage`, `codecovUrl`, and `statusCode` are duplicated in `result` and `result["codecov"]` for now. This duplication will be removed in a followup PR after the FE that depends on `result` is changed